org.pgadmin.pgadmin4.yml

```diff
@@ -118,7 +118,7 @@ modules:
         -xf -
       - cp -r usr/* ${FLATPAK_DEST}/
       - mv ${FLATPAK_DEST}/pgadmin4 ${FLATPAK_DEST}/pgAdmin4
-      - cat config_distro.py usr/pgadmin4/web/config_distro.py > ${FLATPAK_DEST}/pgAdmin4/web/config_distro.py
+      - cat usr/pgadmin4/web/config_distro.py config_distro.py > ${FLATPAK_DEST}/pgAdmin4/web/config_distro.py
```

Comment:
Why the change in order?

Reply:
distro config overrides some of the defaults